src/site/content/en/authors/feed.njk

```diff
@@ -30,7 +30,8 @@ pagination:
   <logo>{{ renderData.metadata.logo }}</logo>
   <subtitle>{{ renderData.metadata.subtitle }}</subtitle>
   {% for post in paged.elements -%}
-  {% set absolutePostUrl -%}{{ post.url | stripLanguage | url | absoluteUrl(renderData.metadata.url) }}{%- endset %}
+  {% set postUrl = '/' + post.fileSlug + '/' %}
+  {% set absolutePostUrl -%}{{ postUrl | url | absoluteUrl(renderData.metadata.url) }}{%- endset %}
```

Comment:
Do we need [the url filter](https://www.11ty.dev/docs/filters/url/) here? It seems like it might be a no-op because you turned fileSlug into an absolute url in the step before.

Btw would you mind include a comment above that links to the absoluteUrl docs
https://www.11ty.dev/docs/plugins/rss/#supplies-the-following-nunjucks-filters

Just so folks know where that filter comes from.